fcrepo-fixity/src/test/resources/OSGI-INF/blueprint/blueprint-integration-test.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Good point -- at one point they were not identical, but now they can be collapsed into a single file.